tests/io/lobster/test_outputs.py

```diff
@@ -1614,14 +1616,104 @@ def test_has_good_quality(self):
             number_occ_bands_spin_up=1, limit_deviation=1e-8
         )
         assert self.band_overlaps2.has_good_quality_check_occupied_bands(number_occ_bands_spin_up=10, limit_deviation=1)
-        assert not self.band_overlaps2_new.has_good_quality_check_occupied_bands(
+        assert self.band_overlaps2_new.has_good_quality_check_occupied_bands(
             number_occ_bands_spin_up=2, limit_deviation=0.1
         )
         assert self.band_overlaps2.has_good_quality_check_occupied_bands(number_occ_bands_spin_up=1, limit_deviation=1)
         assert self.band_overlaps2_new.has_good_quality_check_occupied_bands(
             number_occ_bands_spin_up=1, limit_deviation=2
         )
 
+    def test_has_good_quality_check_occupied_bands_patched(self):
+        """Test with patched data."""
+        import copy
```

Comment:
Perhaps we don't need to lazy load `copy`?

Reply:
Done

Reply:
Thanks